src/Analyzers/CSharp/Tests/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.cs

```diff
@@ -2552,55 +2733,40 @@ public void Goo(int x = 0)
 
     static void Main()
     {
-        ((I)new C()).Goo();
+        ([|(I)|]new C()).Goo();
     }
 }
-";
-
-            await VerifyCS.VerifyCodeFixAsync(source, source);
-        }
-
-        [WorkItem(34326, "https://github.com/dotnet/roslyn/issues/34326")]
```

Comment:
the diff is terrible.  but new cases were added which github is showing bad diffs with the old tests.